tests/subsys/tracing/tracing_sys/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+tests:
+  tracing.tracing_sys.disable:
```

Comment:
why do we need this exactly?

Reply:
Hi @nashif , I want to verify that tracing does not interfere with the system behavior originally by comparison with tracing enabled. After thinking, maybe the second configure can verify this point. I will correct this. Thanks.